apparmor.d/groups/lxqt/featherpad

```diff
@@ -0,0 +1,57 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2024 Alexandre Pujol <alexandre@pujol.io>
+# Copyright (C) 2024 Besanon  <m231009ts@mailfence.com>
+# SPDX-License-Identifier: GPL-2.0-only
+
+#include <tunables/global>
+
+@{exec_path} = @{bin}/featherpad
+profile featherpad @{exec_path} {
+  include <abstractions/base>
+  include <abstractions/common/app>
+  include <abstractions/gtk>
+  include <abstractions/gvfs-open>
```

Comment:
The standard way to allow opening resource in this project is to use one of the [child-open](https://github.com/roddhjav/apparmor.d/tree/main/apparmor.d/groups/children) profile. Eg: `  @{open_path}           rPx -> child-open,`